ops/docker/ci-builder/Dockerfile

```diff
@@ -26,6 +26,8 @@ RUN source $HOME/.profile && \
 
 FROM ethereum/client-go:alltools-v1.10.25 as geth
 
+FROM ghcr.io/crytic/echidna/echidna:testing-master as echidna-test
```

Comment:
suggest to maybe pin this to a specific version so it doesn't break unexpectedly? otherwise lgtm